pkg/pdp/curio/client.go

```diff
@@ -15,6 +15,20 @@ import (
 	"github.com/storacha/go-ucanto/principal"
 )
 
+type PDPClient interface {
```

Comment:
Defined interface for Curio client, allowing it to be mocked in `curiofinder_test` and `curioadder_test`.